src/librt/primitives/annot/annot.c

```diff
@@ -1181,6 +1185,10 @@ rt_annot_export5(struct bu_external *ep, const struct rt_db_internal *ip, double
 		bu_strlcpy((char *)cp, bu_vls_addr(&tseg->label), bu_vls_strlen(&tseg->label) + 1);
 
 		cp += bu_vls_strlen(&tseg->label) + 1;
+		*(uint32_t*)cp = htonl(tseg->txt_size);
```

Comment:
Looks like the host-to-net ones are here.  Several examples of how to call bu_cv_htond() in annot.c already, but let me know if you need explanation.  The header in include/bu/cv.h should document the usage.